client/state/data-layer/utils.js

```diff
@@ -3,7 +3,16 @@
 /**
  * External dependencies
  */
-import { camelCase, isArray, isObjectLike, isPlainObject, map, reduce, set } from 'lodash';
+import {
+	camelCase,
+	snakeCase,
+	isArray,
+	isObjectLike,
+	isPlainObject,
+	map,
+	reduce,
+	set,
```

Comment:
NIT: I'd love to get these alphabetized, especially since prettier line-breaks them.